CONTRIBUTING.md

```diff
@@ -6,42 +6,87 @@ We're happy you want to contribute! You can help us in different ways:
   suggestions for improvements
 * Fork this repository and submit a pull request
 
-As with the majority of major open source projects, we ask that
-Citus contributors sign a Contributor License Agreement (CLA). We
-know that this creates a bit of red tape, but also believe that it
-is the best way to create total transparency around your rights as
-a contributor and the most effective way of protecting you as well
-as Citus Data as the company behind the open source project.
-
-If you plan on contributing to Citus, please follow these [instructions
-for signing our CLA](https://www.citusdata.com/community/CLA)
+Before accepting any code contributions we ask that Citus contributors
+sign a Contributor License Agreement (CLA). For an explanation of
+why we ask this as well as instructions for how to proceed, see the
+[Citus CLA](https://www.citusdata.com/community/CLA).
 
 ### Getting and building
 
 #### Mac
 
-1. Install XCode
+1. Install Xcode
 2. Install packages with homebrew
 
    ```bash
    brew update
-   brew install git openssl postgresql
-   brew link openssl --force
+   brew install git postgresql
    ```
 
-3. Get the code
+3. Get, build, and test the code
 
    ```bash
    git clone https://github.com/citusdata/citus.git
+
+   cd citus
+   ./configure
+   make
+   make install
+   cd src/test/regress
+   make check
+   ```
+
+#### Debian-based Linux (Ubuntu, Debian)
+
+1. Install build dependencies
+
+   ```bash
+   echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" | \
+        sudo tee /etc/apt/sources.list.d/pgdg.list
+   wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
+        sudo apt-key add -
+   sudo apt-get update
+
+   sudo apt-get install -y postgresql-server-dev-9.5 postgresql-9.5  \
+                libedit-dev libselinux1-dev libxslt-dev libpam0g-dev \
+                git flex make
    ```
 
-4. Build and test
+2. Get, build, and test the code
 
    ```bash
+   git clone https://github.com/citusdata/citus.git
    cd citus
    ./configure
    make
    sudo make install
    cd src/test/regress
-   make check-multi
+   make check
+   ```
+
+#### Red Hat-based Linux (RHEL, CentOS, Fedora)
+
+1. Find the PostgreSQL 9.5 RPM URL for your repo at [yum.postgresql.org](http://yum.postgresql.org/repopackages.php#pg95)
+2. Register its contents with Yum:
+
+   ```bash
+   sudo yum install -y <url>
+   ```
+
+3. Install build dependencies
+
+   ```bash
+   sudo yum -y update
+   sudo yum groupinstall -y 'Development Tools'
+   sudo yum install -y \
```

Comment:
Hm. Well, whatever you do with the package wrapping, make the Red Hat and Debian sections match formatting. This is different from above.